server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,19 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+async function checkWorkerAvailability(workerTimeslotId) {
+    //Selects all from workerTimeslot table 
+    const queryStatement = `select * from ${Tables.workerTimeslot}  where worker_timeslot_id = ${workerTimeslotId}`;
+    const queryOutput = await db.any(queryStatement);
+    //check if workertimeSlot is present 
+    if (_.isEmpty(queryOutput)) return false;
+    //check if available 
+    const currentStatus = _.includes(queryOutput, 'status'); 
```

Comment:
Hi Nancy, 
We would still need to map the output to their status values. Thus, I would recommend keeping this line as `_.map(...)`.

Reply:
Hi Amy,
Thank you for your suggestion!
My Action: I have replaced the lodash `includes` with `map`

You can see this change in line 31 of server/models/db/workerTimeslot.js

Please confirm if this takes care of your suggestion!